conceptnet5/readers/nadya.py

```diff
@@ -53,7 +55,7 @@ def handle_line(line, builder):
     """
     parts = line.rstrip('\n').split('\t')
     (cnet4_id, lang, frame_text, relname, start_text, end_text,
-     freq, vote, creator, voter) = parts
+     freq, vote, email, creator, voter) = parts
```

Comment:
`vote` is cast to an int in 65, but never used.

Reply:
Good catch -- I actually should be checking that the vote is positive.